information_requests/models.py

```diff
@@ -91,6 +95,9 @@ def __str__(self):
 
 
 class Complaint(CreatedUpdatedAtMixin, models.Model):
+    user_complaint = models.ForeignKey(
```

Comment:
Pode ser só user aqui também.
```suggestion
    user = models.ForeignKey(
```